src/shared/types/data/Matchs.ts

```diff
@@ -54,5 +48,6 @@ interface LeagueOfLegendsGame {
       }[];
     }
   >;
+  duration: string;
```

Comment:
The duration should be a number of seconds, and it is formatted inside the component.